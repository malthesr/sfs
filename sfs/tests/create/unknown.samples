sample0
sample1
sample1000
