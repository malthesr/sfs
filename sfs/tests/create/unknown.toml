args = [
  "create",
  "--allow-stdin",
  "-S",
  "tests/create/unknown.samples",
  "tests/create/simple.bcf"
]
status = "failed"
