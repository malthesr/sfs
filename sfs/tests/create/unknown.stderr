[sfs  warn] sample sample1000 was not found in input file
